apps/jobs/views.py

```diff
@@ -1488,11 +1488,9 @@ def get_bearer_token(request, challenge_pk):
 
 
 @api_view(["GET"])
-@throttle_classes([UserRateThrottle])
-@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
-@authentication_classes((ExpiringTokenAuthentication,))
+@throttle_classes([AnonRateThrottle])
 def get_github_badge_data(
-    request, challenge_phase_split_pk, participant_team_pk
+    request, challenge_phase_split_pk, participant_team_name
```

Comment:
Happy with this switch from participant_team_pk to participant_team_name but just curious what was the reasoning behind changing this?